src/data/eventFiltering.js

```diff
@@ -35,6 +35,5 @@ export function populateEventFilterFrom(queryStringData) {
 }
 
 export function isParticipant(event) {
-  // `has_joined` is not always provided, so we should explicitly check against `true`
-  return event.has_joined === true
+  return !!event.has_joined
```

Comment:
Seems like this function could be removed and this statement placed above, since the function only contains one statement.

Reply:
In this case, I think it provides a bit of value as self-documenting code. It also serves as a wrapper in case the backend implementation changes (again). I would be happy with it either way, though.

Reply:
Alright, sounds good.